Test_API_Login/Test_API_Login.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  Test_API_Login.swift
+//  Test_API_Login
+//
+//  Created by Cristian Sancricca on 27/05/2022.
+//
+
+
+import XCTest
+@testable import SwiftTemplate
+
+class Test_API_Login: XCTestCase {
+
+    var service: LoginService!
```

Comment:
es una convencion que el elemento que vayamos a testear se le ponga de nombre "sut" System Under Test 
https://stackoverflow.com/questions/7321407/what-is-sut-and-where-did-it-come-from